ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransactions.java

```diff
@@ -130,11 +134,11 @@ public void evictOldTransactions() {
   public boolean addRemoteTransaction(final Transaction transaction) {
     final TransactionInfo transactionInfo =
         new TransactionInfo(transaction, false, clock.instant());
-    final boolean transactionAdded = addTransaction(transactionInfo);
-    if (transactionAdded) {
+    final TransactionAddedStatus transactionAddedStatus = addTransaction(transactionInfo);
```

Comment:
We can use a variable to avoid doing the equals twice

Reply:
Done.